cyhy/mailer/CyhyNotificationMessage.py

```diff
@@ -32,14 +32,18 @@ class CyhyNotificationMessage(ReportMessage):
     TextBody = """Greetings {{acronym}},
 
 Cyber Hygiene scans of your host(s) conducted in the past day have detected one or both of the following:
```

Comment:
With how the wording here has grown, I think this sounds better, don't you?
```suggestion
Cyber Hygiene scans of your host(s) conducted in the past day have detected one or more of the following:
```

Reply:
Makes sense to me! :)